modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -82,6 +119,24 @@ impl SbomContext {
                 package.package_version.clone(),
                 refs,
             );
+
+            if product_packages.contains(&package.package_spdx_identifier) {
+                let pr = self
+                    .graph
+                    .ingest_product(
+                        package.package_name.clone(),
+                        ProductInformation {
+                            vendor: package.package_supplier.clone(),
+                        },
+                        (),
```

Comment:
Shouldn't that be part of the transaction?

Reply:
Fixed. Thanks for spotting it